src/portal/lib/src/config/project-quotas/project-quotas.component.scss

```diff
@@ -0,0 +1,47 @@
+.default-quota {
+  display: flex;
+
+  .default-quota-text {
+    display: flex;
+    justify-content: space-between;
+    min-width: 13rem;
+    padding-right: 1rem;
+
+    .num-count {
+      display: inline-block;
+      min-width: 2rem;
+    }
+  }
+}
+
+.padding-top {
+  padding-top: 1rem;
+  color: #000;
+}
+
+.progress-block {
+  label {
+    font-weight: 400 !important;
+  }
+}
+
+.default-quota-edit-button {
+  margin-top: 0;
+  padding-top: 0;
+  height: 1rem;
+}
+
+.min-label-width {
+  min-width: 120px;
+}
+
+.quota-top {
+  display: flex;
+  justify-content: space-between;
+}
+
+.refresh-div {
+  margin-top: auto;
+  margin-right: 1rem;
+  cursor: pointer;
+}
```

Comment:
there is many normal css class provided by Clarity。eg: "mr-1":margin-right: 1rem, "pt-1": padding-top:1rem and so on,you can use them directly.